frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -30,6 +59,11 @@ export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): bool
   return independent === 'true';
 };
 
+export const getRequiredKeys = (csv: ClusterServiceVersionKind): string[] => {
+  const keys = csv.metadata.annotations?.['external.cluster.ocs.openshift.io/keys'];
```

Comment:
type of keys?

Reply:
string(implicit)

Reply:
can you make it explicit?

Reply:
The only reason to make it explicit would be if the implicit type is `any`. Here the implicit type is correct (`string`) since the type is properly defined in `ClusterServiceVersionKind`.